src/features/auth/components/register-form.tsx

```
@@ -0,0 +1,107 @@
+                    />
+                    <span className='text-red-600 text-sm'>
+                        {formState.errors.studentId ? (
+                            formState.errors.studentId?.message
+                        ) : ''}
+                    </span>
+                    <span className="text-label-1 mb-1 mt-2.5 text-[#3d3d3d]">전화번호</span>
+                    <input
+                        className={`border-1 mb-0.5 rounded-sm p-2 px-2.5 ${formState.errors.phoneNumber ? 'border-red-600 placeholder-red-600 text-red-600' : 'border-[#888888] placeholder-[#888888]'
+                            }`}
+                        {...register('phoneNumber')}
+                        type="phone"
```

ESLint

[error] Delete `············`

(prettier/prettier)